src/SentryOne.UnitTestGenerator/Commands/GenerationItem.cs

```diff
@@ -41,7 +41,20 @@ public GenerationItem(ProjectItemModel source, SyntaxNode sourceSymbol, ProjectI
 
         public string TargetContent { get; set; }
 
-        public string TargetFileName => Path.Combine(_targetPath, _options.GetTargetFileName(Path.GetFileNameWithoutExtension(Source.FilePath)) + Path.GetExtension(Source.FilePath));
+        public string TargetFileName
+        {
+            get
+            {
+                var targetFileName = _options.GetTargetFileName(Path.GetFileNameWithoutExtension(Source.FilePath)) + Path.GetExtension(Source.FilePath);
+                if (string.IsNullOrEmpty(_targetPath))
```

Comment:
Use string.IsNullOrWhitespace here?

Reply:
Have reviewed, is ok as NullOrEmpty in our usage